README.dm

```diff
@@ -0,0 +1,40 @@
+# Hello World
+
+> This is my first project as a Microverse student.
+
+![screenshot](image/SS-hello-world.png)
+
+In this project I made this repository with my new knowledge that I learned in my second day at Microverse.
+
+## Built With
+
+- HTML & CSS
+- Visual Studio Code
+
+
+## Author 👤   
+
+
+
+- GitHub: [@githubhandle](https://github.com/AlexRS90)
+- Twitter: [@twitterhandle](https://twitter.com/AlejandroRBenji)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/alejandro-ramos-santos-9b0b52135/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] [Optional] it would be nice if you include a valid link for issues 🤝

Reply:
Hello Daniel!!
Yes, of course 🤓 
Now it's done 😄